hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoDeleteHandlerTest.java

```diff
@@ -77,7 +81,10 @@ void preHandlesCryptoDeleteIfReceiverSigRequiredVanilla() {
 
         final var txn = deleteAccountTransaction(deleteAccountId, transferAccountId);
 
-        final var meta = subject.preHandle(txn, payer, store);
+        final var builder =
+                new SigTransactionMetadataBuilder(store).txnBody(txn).payerKeyFor(payer);
+        subject.preHandle(builder);
+        final var meta = builder.build();
```

Comment:
```
final var builder =
                new SigTransactionMetadataBuilder(store).txnBody(txn).payerKeyFor(payer);
        subject.preHandle(builder);
        final var meta = builder.build();
```
This is done (with different params that often that a method would really make sense)

Reply:
Like `prehandleAndBuildMetadata(store, txn, payer, subject);`